# Makes possible to have _JAVA_OPTIONS
# ----
# If _JAVA_OPTIONS is defined, the `java -version` command will print the
# version of Java in the second line, not the first
#
# Ignores the jfxrt.jar test
# ----
# The program would exit if jfxrt.jar existed
--- a/build.gradle	Fri Dec 12 16:17:47 2014 -0800
+++ b/build.gradle	Mon Dec 29 22:40:58 2014 +0000
@@ -726,6 +726,9 @@
 try {
     if (inStream.readLine() != null) {
         String v = inStream.readLine();
+        while (!v.contains(" Runtime")) {
+            v = inStream.readLine();
+        }
         if (v != null) {
             int ib = v.indexOf(" (build ");
             if (ib != -1) {
@@ -1063,12 +1066,14 @@
 
 // Task to check whether jfxrt.jar is present in the JDK
 task checkJfxrtJar {
+    /*
     doLast {
         def jfxrtFile = new File("$JDK_HOME/jre/lib/ext/jfxrt.jar");
         if (jfxrtFile.exists()) {
             fail("$jfxrtFile must be removed before building sdk")
         }
     }
+    */
 }
